modes/lms/evil-collection-lms.el

```diff
@@ -0,0 +1,209 @@
+  (kill-buffer)
+  (lms-ui-playing-now-refresh))
+
+(defalias 'evil-collection-lms-ui-year-album-artist-list-mode-quit
+  'evil-collection-lms-ui-tracks-list-mode-quit
+  "Exit lms-ui-tracks-list-mode.")
+
+(defun evil-collection-lms-goto-bottom ()
+  "Move the point to the last penultimate line of the current buffer."
+  (goto-char (max-char))
+  (forward-line -1))
+
+(defun evil-collection-lms-next-line ()
+  "Move the point to the next line, except if it is at the end of the buffer."
+  (interactive)
+  (forward-line 1)
+  (when (eobp) (forward-line -1)))
+
+
+;;;###autoload
+(defun evil-collection-lms-setup ()
+  "Set up `evil' bindings for `lms'."
+
+  (dolist (map evil-collection-lms-maps)
+    (evil-collection-inhibit-insert-state map))
+
+  (dolist (mode evil-collection-lms-modes)
+    (evil-set-initial-state mode 'normal))
+
+  (evil-collection-define-key 'normal 'lms-ui-playing-now-mode-map
+    "t"              'lms-ui-playing-now-change-player-power-state
+    (kbd "C-p")      'lms-ui-playing-now-players-list
+    "R"              'lms-ui-playing-now-change-rating
+    (kbd  "C-=")     'lms-ui-playing-now-set-volume
+    "x"              'lms-ui-playing-now-play-pause
+    (kbd "RET")      'lms-ui-playing-now-play-pause
+    "p"              'lms-ui-playing-now-play
+    "s"              'lms-ui-playing-now-stop
+    "J"              'lms-ui-playing-now-next
+    (kbd  "<right>") 'lms-ui-playing-now-next
+    "K"              'lms-ui-playing-now-prev
+    (kbd  "<left>")  'lms-ui-playing-now-prev
+    "+"              'lms-ui-playing-now-volume-up
+    "="              'lms-ui-playing-now-volume-up
+    "-"              'lms-ui-playing-now-volume-down
+    "m"              'lms-ui-playing-now-volume-mute
+    "r"              'lms-ui-playing-now-cycle-repeat
+    "S"              'lms-ui-playing-now-cycle-shuffle
+    "u"              'lms-ui-playing-now-refresh
+    "i"              'lms-ui-playing-now-show-track-info
+    "P"              'lms-ui-playing-now-show-playlist
+    "T"              'lms-ui-playing-now-album-tracks-list
+    "A"              'lms-ui-playing-now-artist-albums-list
+    "Y"              'lms-ui-playing-now-year-albums-list
+    "M"              'lms-ui-playing-now-browse-music-library
+    "g?"             'lms-ui-playing-now-help
+    "gh"             'lms-ui-playing-now-help
+    "q"              'lms-ui-playing-now-quit)
+
+  (evil-collection-define-key 'normal 'lms-ui-track-info-mode-map
+    "R"             'lms-ui-track-info-change-rating
+    "K"             'lms-ui-track-info-prev
+    (kbd "<left>")  'lms-ui-track-info-prev
+    "J"             'lms-ui-track-info-next
+    (kbd "<right>") 'lms-ui-track-info-next
+    "g?"            'lms-ui-playing-now-help
+    "gh"            'lms-ui-playing-now-help
+    "q"             'evil-collection-lms-ui-track-info-mode-quit)
+
+  (evil-collection-define-key 'normal 'lms-ui-players-mode-map
+    [remap evil-goto-line] 'evil-collection-lms-goto-bottom
```

Comment:
Since it remaps `evil-goto-line` to `evil-collection-lms-goto-bottom`, it should take care of the prefix numeric arg.